infra/core/database/sqlserver/sqlserver.bicep

```diff
@@ -0,0 +1,130 @@
+param name string
```

Comment:
Since it is has been a while and infra core has changed in azure-dev, please do a diff with latest in azure-dev and see what has changed, and merge any changes.